site/source/tutorials/001_static_content/index.md

```diff
@@ -59,7 +58,7 @@ The `v` function simply instructs Dojo 2 to create a HTML element, in this case
 {% solution showSolution1 %}
 ```typescript
 export default class HelloWorld extends WidgetBase<WidgetProperties> {
-	render() {
+	render(): DNode {
```

Comment:
If we're making the change to the type, we should probably also make the change to specify that it is protected while we're at it.